src/components/AvatarImage/AvatarImage.module.css

```diff
@@ -5,9 +5,10 @@
 /**
  * Avatar image SVG
  * 1) This component contains an outer wrapper
- *    and SVG fragments for avatars
+ *    and SVG fragments for avatars.
```

Comment:
Thanks for adding the periods at the end of these comments btw ❤️ for some reason the lack of punctuation always distracts me 😅